hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/connector/jdbc/JdbcSqlConnector.java

```diff
@@ -75,24 +75,22 @@ public String typeName() {
     @Nonnull
     @Override
     public String defaultObjectType() {
-        return "Table";
+        return JdbcDataConnection.OBJECT_TYPE_TABLE;
     }
 
     @Nonnull
     @Override
     public List<MappingField> resolveAndValidateFields(
             @Nonnull NodeEngine nodeEngine,
-            @Nonnull Map<String, String> options,
-            @Nonnull List<MappingField> userFields,
-            @Nonnull String[] externalName,
-            @Nullable String dataConnectionName,
-            @Nullable String objectType) {
-        if (dataConnectionName == null) {
+            @Nonnull SqlExternalResource sqlExternalResource,
```

Comment:
Maybe name the parameter here and in `createTable` only `externalResource` (in all subclasses) - you already do in _some_ places and it may save some ugly statement's spanning multiple lines.

Reply:
90d46a211e6719deeb2b69fc20c3deefafaa78b4